main/src/libraries/wrdac/src/knowledge/object.cpp

```diff
@@ -164,6 +170,7 @@ bool Object::fromBottle(const Bottle &b)
     m_color[1] = b.find(EFAA_OPC_OBJECT_GUI_COLOR_G).asDouble();
     m_color[2] = b.find(EFAA_OPC_OBJECT_GUI_COLOR_B).asDouble();
     m_saliency = b.find(EFAA_OPC_OBJECT_SALIENCY).asDouble();
+    m_value = b.find(EFAA_OPC_OBJECT_VALUE).asDouble();
```

Comment:
The `b.check(EFAA_OPC_OBJECT_VALUE)` should be added above to check whether this value is present in the bottle.